docs/customer-360/customer-journey/process.sql

```diff
@@ -0,0 +1,24 @@
+SET 'auto.offset.reset' = 'earliest';
+
+-- Create stream of pages
+CREATE STREAM pages (
+  customer INTEGER,
+  time BIGINT,
+  page_id STRING,
+  page STRING
+) WITH (
+  VALUE_FORMAT = 'JSON',
+  KAFKA_TOPIC = 'pages',
+  PARTITIONS = 6
+);
+
+-- Create stateful table with list of pages visited by each customer
+-- Pages are added to an Array using the `COLLECT_LIST` function
+CREATE TABLE pages_per_customer WITH (KAFKA_TOPIC = 'pages_per_customer') AS
+SELECT
+  customer,
+  COLLECT_LIST(page) AS page_list,
```

Comment:
@MichaelDrogalis this recipe uses `COLLECT_LIST` as we discussed in a separate forum.  Do you think this set of ksqlDB statements is good enough, or any suggestions on how we can extend this recipe in more interesting ways

Reply:
Could also be interesting to include a `COUNT_DISTINCT` on the number of pages someone's visited, too. But LGTM!

Reply:
@MichaelDrogalis great idea, added in latest commit